foreshadow/tests/test_foreshadow.py

```diff
@@ -1050,3 +1050,36 @@ def test_foreshadow_serialization_tpot():
     # converge. The test here aims to evaluate if both cases have
     # produced a reasonable score and the difference is small.
     assert score1 > 0.9 and score2 > 0.9
+
+
+def test_foreshadow_configure_sampling():
```

Comment:
for this test case, maybe copy an existing integration test case but enable sampling to verify the sampling logic is working.

Reply:
Done.